### PR TITLE
AWS Blockchain Node Runners

### PR DESCRIPTION
**Pull request by varnarmat**
## Description

- I added documentation (and two complimentary architectures) for how to deploy Besu Teku Ethereum Nodes via AWS Blockchain Node Runners Blueprint for Ethereum in order to showcase a self managed way in which you can deploy your Ethereum Nodes on AWS (without using Amazon Managed Blockchain). AWS Blockchain Node Runners is an open-source project that provides developers with a set of TypeScript CDK blueprints. These blueprints can be used to deploy nodes for various blockchain protocols, including Ethereum, on AWS infrastructure.

### Issue(s) fixed

No issues

Fixes #

### Preview

- https://besu-docs-4psp4fuiu-hyperledger.vercel.app/public-networks/how-to/deploy-aws-node-runners

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
@macfarla or @siladu Would you be able to verify/ approve the changes made in this PR?

**Review by alexandratran — APPROVED** [review]
Added major edits to the content to meet documentation standards. LGTM but will seek additional review.

File: docs/public-networks/how-to/deploy-aws-node-runners.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+    It is not available from the Internet and is not open for external connections.
+    Before opening it up, protect your RPC APIs.
+    :::
+
+### 4. Clear and undeploy nodes
+
+To clear and undeploy the RPC nodes, sync nodes, and common components, use the following commands:
+
+```bash
+# Set the AWS account ID and region in case the local .env file is lost.
```
Comment (bgravenorst) [inline docs/public-networks/how-to/deploy-aws-node-runners.md:402]: These comments should probably be steps.

**Review by bgravenorst — APPROVED** [review]
Added some feedback

**Comment by macfarla** [comment]
> @macfarla or @siladu Would you be able to verify/ approve the changes made in this PR? Should these be added to the docs?

@varnarmat thanks for your contrib - super detailed! I'm actually reluctant to add this level of detail to the docs directly. Maybe it's better as a guide in wiki or similar? @siladu - thoughts?

**Comment by siladu** [comment]
Thanks for the contribution @varnarmat. No strong opinion on docs vs wiki, but I'd be fine with it as part of the docs. 

I think this page is better suited under the Tutorials section, alongside [Deploy Besu using Kubernetes](https://besu-docs-4psp4fuiu-hyperledger.vercel.app/public-networks/tutorials/kubernetes).

I am curious what the intended audience/users are for this? Presumably public/mainnet Ethereum, but what sort of node operator, individuals or institutions for example?

**Comment by varnarmat** [comment]
> Thanks for the contribution @varnarmat. No strong opinion on docs vs wiki, but I'd be fine with it as part of the docs.
> 
> I think this page is better suited under the Tutorials section, alongside [Deploy Besu using Kubernetes](https://besu-docs-4psp4fuiu-hyperledger.vercel.app/public-networks/tutorials/kubernetes).
> 
> I am curious what the intended audience/users are for this? Presumably public/mainnet Ethereum, but what sort of node operator, individuals or institutions for example?

Thank you. I am open to where we would to put the AWS Blockchain Node Runners (Node Runners) documentation. Node Runners is experimental in nature and is intended for non-production use in proof of connects and pilots.

**Pull request merged**